class.jetpack-gutenberg.php

```diff
@@ -0,0 +1,175 @@
+		wp_enqueue_script( 'jetpack-blocks-view', $view_script, array(), $version );
+		wp_enqueue_style( 'jetpack-blocks-view', $view_style, array(), $version );
+		wp_add_inline_script(
+			'wp-i18n',
+			'wp.i18n.setLocaleData( ' . Jetpack::get_i18n_data_json() . ', \'jetpack\' );'
+		);
+	}
+
+	/**
+	 * Load Gutenberg editor assets
+	 *
+	 * @since 6.7.0
+	 *
+	 * @return void
+	 */
+	public static function enqueue_block_editor_assets() {
+		if ( ! self::should_load_blocks() ) {
+			return;
+		}
+
+		$rtl = is_rtl() ? '.rtl' : '';
+
+		/**
+		 * Filter to enable serving blocks via CDN
+		 *
+		 * CDN cache is busted once a day or when Jetpack version changes. To customize it:
+		 *   add_filter( 'jetpack_gutenberg_cdn_cache_buster', function( $version ) { return time(); }, 10, 1 );
+		 *
+		 * @since 6.5.0
+		 *
+		 * @param bool false Whether to load Gutenberg blocks from CDN
+		 */
+		if ( apply_filters( 'jetpack_gutenberg_cdn', false ) ) {
+			$cdn_base      = 'https://s0.wp.com/wp-content/mu-plugins/jetpack/_inc/blocks';
+			$editor_script = "$cdn_base/editor.js";
+			$editor_style  = "$cdn_base/editor$rtl.css";
+
+			/**
+			 * Filter to modify cache busting for Gutenberg block assets loaded from CDN
+			 *
+			 * @since 6.7.0
+			 *
+			 * @param string
+			 */
+			$version = apply_filters( 'jetpack_gutenberg_cdn_cache_buster', sprintf( '%s-%s', gmdate( 'd-m-Y' ), JETPACK__VERSION ) );
+		} else {
+			$editor_script = plugins_url( '_inc/blocks/editor.js', JETPACK__PLUGIN_FILE );
+			$editor_style  = plugins_url( "_inc/blocks/editor$rtl.css", JETPACK__PLUGIN_FILE );
+			$version       = Jetpack::is_development_version() && file_exists( JETPACK__PLUGIN_DIR . '_inc/blocks/editor.js' )
+				? filemtime( JETPACK__PLUGIN_DIR . '_inc/blocks/editor.js' )
+				: JETPACK__VERSION;
+		}
+
+		wp_enqueue_script(
+			'jetpack-blocks-editor',
+			$editor_script,
+			array(
+				'lodash',
+				'wp-api-fetch',
+				'wp-blocks',
+				'wp-components',
+				'wp-compose',
+				'wp-data',
+				'wp-date',
+				'wp-editor',
+				'wp-element',
+				'wp-hooks',
+				'wp-i18n',
+				'wp-keycodes',
+				'wp-plugins',
+				'wp-token-list',
+			),
+			$version
+		);
+
+		wp_localize_script(
+			'jetpack-blocks-editor',
+			'Jetpack_Block_Assets_Base_Url',
+			plugins_url( '_inc/blocks/', JETPACK__PLUGIN_FILE )
+		);
+
+		wp_enqueue_style( 'jetpack-blocks-editor', $editor_style, array(), $version );
+	}
+
+	/**
+	 * Check whether conditions indicate Gutenberg blocks should be loaded
+	 *
+	 * Loading blocks is enabled by default and may be disabled via filter:
+	 *   add_filter( 'jetpack_gutenberg', '__return_false' );
+	 *
+	 * @since 6.7.0
+	 *
+	 * @return bool
+	 */
+	public static function should_load_blocks() {
+		if ( ! Jetpack::is_active() ) {
+			return false;
+		}
+
+		/**
+		 * Filter to disable Gutenberg blocks
+		 *
+		 * @since 6.5.0
+		 *
+		 * @param bool true Whether to load Gutenberg blocks
+		 */
+		if ( ! apply_filters( 'jetpack_gutenberg', true ) ) {
```

Comment:
```suggestion
		return apply_filters( 'jetpack_gutenberg', true );
```

Reply:
I actually wrote that then changed it 🙂 

Maybe with a cast:

```php
		return (bool) apply_filters( 'jetpack_gutenberg', true );
 ```

Reply:
or a `! !` :)

Reply:
✅  d24aaca73dbbf615cede3fdf48cb9c38b9c7ae80